### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by christopherasnider**
Added:
- Uploader 2.11.0
- Web 1.16.0, 1.16.1, 1.17.0

### CODE REVIEW

**Comment by christopherasnider** [comment]
I tried to avoid getting too clever with the bug details across the board.

File: _docs/release-notes.md [hunk 1]
```diff
@@ -24,6 +24,50 @@ Visit [https://app.tidepool.org](https://app.tidepool.org) to see the results of
 
 Don't have a Tidepool account? Visit [tidepool.org/signup](https://tidepool.org/signup) to create your free Tidepool account.  
 
+### 1.17.0 (Released 2019-03-05)
+There was a data paging bug that was triggered in instances where a large gap of time without data is followed by a data point that is diabetes-specific (like a data point) rather than device-related (like an insulin pump event). In such cases, data that occurs before the gap would not be segmented and loaded (we call this paging), but instead would be fetched and processed all at once. Depending on how much data exists before the gap, the browser will either slow or crash. Which is a bummer, so we fixed that.  
+[Feature Requirements](https://trello.com/c/6tfqWKVM/6-v1170-some-user-accounts-processing-all-fetched-data-at-once)
+
+Carb events not associated with a bolus are now displayed in Tidepool Web. Here's looking at you, people entering carbs as Dexcom events or folks uploading carb data via Apple Health.
```
Comment (gniezen) [inline _docs/release-notes.md]: At the moment carb data from devices uploaded with the uploader will not be displayed, as the work to ingest carb events, notes etc. is still to come. So it is only for carb data coming in through Tidepool Mobile at this stage.

File: _docs/release-notes.md [hunk 2]
```diff
@@ -24,6 +24,50 @@ Visit [https://app.tidepool.org](https://app.tidepool.org) to see the results of
...
+[Code Update](https://github.com/tidepool-org/blip/releases/tag/v1.17.0)
+
+### 1.16.1 (Released 2019-03-01)
+Coming in hot! We noticed this after releasing v1.16.0: when logged in as a clinician, the scrollbar in the patient list disappeared. I'm told by our engineers this was a 2-line fix. Sounds simple enough.  
+[Feature Requirements](https://trello.com/c/EMApNn9D/2-regression-scrollbar-missing-from-vca-patients-list)
+
+[Code Update](https://github.com/tidepool-org/blip/releases/tag/v1.16.1)
+
+### 1.16.0 (Released 2019-02-28)
+This is a big one, folks. Tidepool Web now shows a bunch of new aggregated statistics including more detailed time in range calculations, basal/bolus breakdowns, sensor usage, standard deviation, coefficient of variation, and a Glucose Management Indicator (GMI). It's quite lovely, and a hefty design and engineering effort. Here's what went down.
```
Comment (gniezen) [inline _docs/release-notes.md]:
```suggestion
This is a big one, folks. Tidepool Web now shows a bunch of new aggregated statistics including more detailed time-in-range calculations, basal/bolus breakdowns, sensor usage, standard deviation, coefficient of variation, and a Glucose Management Indicator (GMI). It's quite lovely, and a hefty design and engineering effort. Here's what went down.  
```

File: _docs/release-notes.md [hunk 3]
```diff
@@ -468,6 +512,23 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
 
 If you already have the Tidepool Uploader installed on your computer, it will automatically update to the latest version.  
 
+### 2.11.0 (Released 2019-02-20)
+It's another bug bash bonanza!
+
+Insulin on board (IOB) for Omnipod uploads was miscalculated when either the meal or correction IOB was zero. That's been fixed.
```
Comment (gniezen) [inline _docs/release-notes.md]:
```suggestion
Insulin on board (IOB) for Omnipod uploads was not shown when either the meal or correction IOB was zero. That's been fixed.  
```

File: _docs/release-notes.md [hunk 4]
```diff
@@ -468,6 +512,23 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
...
 If you already have the Tidepool Uploader installed on your computer, it will automatically update to the latest version.  
 
+### 2.11.0 (Released 2019-02-20)
+It's another bug bash bonanza!
+
+Insulin on board (IOB) for Omnipod uploads was miscalculated when either the meal or correction IOB was zero. That's been fixed.  
+[Feature Requirements](https://trello.com/c/tWxKUIcy/17-v2110-omnipod-iob-possibly-miscalculated)
+
+We also addressed instances where device settings on Medtronic 5/7 series pumps were wonky and unexpected. Wonky is a technical term, btw.  
+[Feature Requirements](https://trello.com/c/tWxKUIcy/17-v2110-omnipod-iob-possibly-miscalculated)
```
Comment (gniezen) [inline _docs/release-notes.md]: Links to wrong Trello card.

Comment (christopherasnider) [reply _docs/release-notes.md]: Good catch.

File: _docs/release-notes.md (hunk 1 above)
Comment (christopherasnider) [reply _docs/release-notes.md]: I think I got over-ambitious because the Trello card mentioned the Dexcom API. This will be revised in the updated PR.

**Comment by christopherasnider** [comment]
I think I got all of the changes incorporated. GitHub is still kinda foreign to me...

File: _docs/release-notes.md (hunk 1 above)
Comment (gniezen) [reply _docs/release-notes.md]: I'm sorry, you're right! If coming through the Dexcom API the carb events will be recorded. So how do we make it clear that it works except when you're using the Uploader (for now)?

Comment (christopherasnider) [reply _docs/release-notes.md]: A quick mention should do the trick. Check the latest edit.

**Comment by christopherasnider** [comment]
Updated text clarifying which scenarios will display independent carbs. @gniezen

**Comment by gniezen** [comment]
Excellent, thanks Christopher!

**Pull request merged**